index.html

```diff
@@ -5,8 +5,184 @@
     <title>HTML Academy: Техномарт</title>
   </head>
   <body>
+    <header>
+      <section>
+        <a href="index.html">
+           <img src="img/index-logo.svg"></a>
+        <ul>
+          <li><form></form></li>
+          <li><a href="">Закладки </a></li>
+          <li><a href="">Корзина</a></li>
+          <li><a href="">Оформить заказ</a></li>
+        </ul>
+      </section>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
+      <section>
+        <ul>
+           <li>Интернет магазин<br>строительных материалов<br>и инструментов для ремонта</li>
+           <li>+7(812)555-05-55<br>г.Санкт-Петербург,ул. Б.Конюшенная,д.19/8</li>
+           <li>Войти</li>
+           <li>Регистрация</li>
+        </ul> 
+      </section>
+     <nav>
+        <ul>
+          <li><a href="index.html">Главная</a></li>
```

Comment:
```suggestion
          <li><a>Главная</a></li>
```
так как это текущая страница.